tests/core/test_i64.in

```diff
@@ -0,0 +1,29 @@
+#include <stdio.h>
```

Comment:
OK; I'm fine with leaving things for future changes. I do think that having the files properly named would probably make some other things better too, e.g. the `force_c` flag for `do_run_from_file`